src/azure-cli/azure/cli/command_modules/profile/_help.py

```diff
@@ -55,33 +77,54 @@
 helps['account show'] = """
 type: command
 short-summary: Get the details of a subscription.
-long-summary: If no subscription is specified, shows the current subscription.
+long-summary: >
+    If no subscription is specified, information for the current subscription is returned.
+examples:
+    - name: >
+        Get the current default subscription returning results as JSON.  Use `--output yaml` for plain text. 
+        You can also get details of the current subscription by using `az account list --query "[?isDefault]"`.
+      text: >
+        az account show --output json
+    - name: Get information about a different subscription returning results as a table.
+      text: >
+        az account show --name "My subscription name" --output table
+    - name: Get information about a different subscription using the subscription ID.
+      text: >
+        az account show --name 00000000-0000-0000-0000-000000000000
+    - name: >
+        Get information about a subscription using `query`.  
+        To see available query options, first run `az account show --output json`.
+      text: >
+        az account show --query name
+        az account show --query user.name
+        az account show --query tenantId
+    - name: Return results in plain text.
+      text: >
+        az account show --query tenantId -o tsv
```

Comment:
Personally, I prefer not to include `--query` examples specific to a command 
(in this case `az account show`) as we already have a general documment for `--query`: https://learn.microsoft.com/en-us/cli/azure/query-azure-cli?tabs=concepts%2Cbash

Reply:
I guess this depends on if we are working to improve knowledge of our querying ability. @dbradish-microsoft, since this is a larger scale product, do you have insights showing we should be improving our content around queries?

Reply:
I feel that for our most used commands, like az account, I have to be careful not to turn our auto-gen content into a tutorial.  :-)  Also, we have a Learn H2 planned so customers can drop down to working How-to guides and tutorials that are more complex in nature.  (Because of this I also feel that we should limit hard-coding HTTPS links in autogen as much as possible.)  Keeping all this in mind, I feel `--query` examples are appropriate for key commands.  There is a balance to be achieved that **leans toward no query examples.**  

@chasewilson, I don't want to replicate [How to query Azure CLI command output using a JMESPath query](https://learn.microsoft.com/en-us/cli/azure/query-azure-cli?view=azure-cli-latest&tabs=concepts%2Cbash) in autogen.  I do want to use `--query` judiciously to help our customers start thinking about using the Azure CLI in multi-step scripts and variables.

Let's revisit this discussion once we get `az accoun`t, `az group1, and `az someService` completed.  I think we'll start to see a good pattern of examples by then, and maybe need to backtrack on `--query` if we think we've taken it too far.  Sound good?